slack_bolt/context/context.py

```diff
@@ -33,94 +42,132 @@ def to_copyable(self) -> "BoltContext":
 
     @property
     def client(self) -> Optional[WebClient]:
-        """The `WebClient` instance available for this request.
+        f"""The `WebClient` instance available for this request.
 
-            @app.event("app_mention")
+            @app.{CLIENT}("app_mention")
```

Comment:
I think this change might be a mistake?

Reply:
I think `@app.{CLIENT}("app_mention")` should render to `@app.client("app_mention")` but I could be wrong, is this what you think the mistake could be?

Reply:
@WilliamBergamin -Should it resolve to "event" though and not to "client"?

Reply:
Ohh wow, completely missed that thanks